packages/elements/src/item/index.ts

```diff
@@ -204,6 +219,44 @@ export class Item extends ControlElement {
     }
   }
 
+
+  /**
+   * Get Item content
+   * @returns return item content from slot or label and sub-label
+   */
+  private getItemContent (): string {
+    if (this.isSlotEmpty) {
+      let text = '';
+      if (this.label) {
+        text += this.label;
+      }
+      if (this.subLabel) {
+        text += ` (${this.subLabel})`;
+      }
+      return text;
+    }
+    else {
+      return this.textContent || '';
```

Comment:
`this.textContent` includes left and right slot content. You want to include label and sublabel only

Reply:
Updated.